pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
For the record, the way how [Kubernetes Pods work](https://github.com/openshift/origin/blob/a0f9bbab241b4476019dde7e02235b3003e8ebe6/Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/dockertools/manager.go#L856-L861), and how our [Deployment Lifecycle Hooks work](https://github.com/openshift/origin/blob/a0f9bbab241b4476019dde7e02235b3003e8ebe6/pkg/deploy/strategy/support/lifecycle.go#L264), is to map Command -> Entrypoint and Args -> Cmd.

The [`Command`](https://github.com/openshift/origin/blob/2245e96e0e3f00f8633f5d9e7adfc62c520aad24/pkg/deploy/api/types.go#L104) field in deployment hooks become the container's `Entrypoint`, and there is no way to specify `Args` (no such field). That means that if you want to use a shell you need to call it explicitly.

I remember the shell inconvenience when trying to use Post Deployment Hook to run "rake db:migrate".